src/sidebar/components/SelectionTabs.tsx

```diff
@@ -125,13 +125,28 @@ function SelectionTabs({
 
   const showNotesUnavailableMessage = selectedTab === 'note' && noteCount === 0;
 
+  const tabCountsSummaryPieces = [];
+  if (annotationCount > 0) {
+    tabCountsSummaryPieces.push(`${annotationCount} annotations`);
+  }
+  if (noteCount > 0) {
+    tabCountsSummaryPieces.push(`${noteCount} notes`);
+  }
+  if (orphanCount > 0) {
+    tabCountsSummaryPieces.push(`${orphanCount} orphans`);
+  }
```

Comment:
Maybe it's worth handling singular/plural forms for `annotation/s`, `note/s` and `orphan/s`, depending on the values being 1 or greater than 1.

I reckon you created a simple [pluralization function](https://github.com/hypothesis/client/blob/bbf1eab4f6ee75afb3eeecaf79e3d88c84d98eb9/src/sidebar/components/ShareDialog/ExportAnnotations.tsx#L259-L262) some time ago that could be used here, but an inline ternary would be also fine, I think.